caffe2/CMakeLists.txt

```diff
@@ -407,8 +409,15 @@ if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
       $<$<BOOL:${INTERN_DISABLE_AUTOGRAD}>:--disable-autograd>
       $<$<BOOL:${SELECTED_OP_LIST}>:--selected-op-list-path="${SELECTED_OP_LIST}">
       --force_schema_registration
+      --gen_lazy_ts_backend
```

Comment:
Any reason this isn't just true by default?

Reply:
yea, I can more easily avoid solving all the build systems at once this way; it lets aros/ovrsource/xplat versions of generate_code continue doing what they used to do for now